content/docs/replication/_index.md

```diff
@@ -74,6 +74,7 @@ the objects still exist in pairs.
 * Start applications after the migration.
 * Replicate `PersistentVolumeClaim` objects within/across clusters.
 * Replication with METRO mode does not need Replicator sidecar and common controller.
+* Different namespaces cannot share the same RDF group for creating volumes with ASYNC mode for PowerMax.
```

Comment:
If we add Async here, let's also inform about the SYNC and METRO here:
* Different namespaces can share the same RDF group for creating volumes with SYNC mode for PowerMax.
* Different namespaces can share the same RDF group for creating volumes with METRO mode for PowerMax.
* Different namespaces cannot share the same RDF group for creating volumes with multiple modes for PowerMax.

Reply:
done.